src/index.ts

```diff
@@ -426,23 +463,23 @@ class Spanner extends GrpcService {
       );
     }
     const formattedName = Instance.formatName_(this.projectId, name);
-    const shortName = formattedName.split('/').pop();
+    const displayName = config.displayName || formattedName.split('/').pop();
     const reqOpts = {
       parent: 'projects/' + this.projectId,
-      instanceId: shortName,
+      instanceId: displayName,
       instance: extend(
         {
           name: formattedName,
-          displayName: shortName,
+          displayName,
+          nodeCount: config.nodes || 1,
```

Comment:
As far as I can tell, the `createInstance` method used to accept the node count to be specified both using the property `nodeCount` as well as `nodes`. If both were set, `nodes` would take precedence over `nodeCount`. This implementation seems to only accept `nodes`, which could break existing client code. Or am I missing something here?

Reply:
As per my comment above, I believe `nodeCount` became a part of `config` by an error when `CreateInstanceRequest` interface was set to extend from `IIinstance` interface. `nodeCount` is not listed in the documentation (which predated the introduction of types), so it is not legacy code. I believe it is confusing to have both `nodes` and `nodeCount` properties.

Reply:
I agree that having both is confusing (and unnecessary), and if it was only recently introduced by the addition of types, then I agree that we should remove it. We should however flag it in the release notes to notify potential users that have used the property in the meantime.

Reply:
Resolving this since we submitted the nodeCount change in the breaking changes PR.